tests/test_dse.py

```diff
@@ -2493,6 +2493,24 @@ def test_hoisting_pow_one(self):
 
         assert len([i for i in FindSymbols().visit(op) if i.is_Array]) == 0
 
+    def test_zeroable_property(self):
+        grid = Grid(shape=(10, 10))
+
+        u = TimeFunction(name='u', grid=grid, space_order=4)
+
+        op = Operator(Eq(u.forward, u.dx.dx))
+        iterations = FindNodes(Iteration).visit(op)
+        v = iterations.pop(2)
+        assert ZEROABLE in v.properties
```

Comment:
Now I'm confused. Shouldn't be `y` the only zeroable here?

Reply:
yes, that's what the assert checks! `v` is `y`

Reply:
I should not do that late. Of course it's y that's what pop does